management/hook_secrets.go

```diff
@@ -0,0 +1,40 @@
+package management
+
+type HookSecrets = map[string]string
+
+type HookSecretsManager struct {
+	*Management
+}
+
+func newHookSecretsManager(m *Management) *HookSecretsManager {
+	return &HookSecretsManager{m}
+}
```

Comment:
I think we should add this to the `HookManager` instead. See the `UserManager` and `RoleManager` for examples.

Reply:
I see your point. I'll move it into the hook manager.